doc/releases/squid.rst

```diff
@@ -13,6 +13,12 @@ v19.2.0 Squid
    68215 <https://tracker.ceph.com/issues/68215>`_ before attempting an upgrade
    to 19.2.0.
 
+   Some users have encountered a Ceph Manager balancer module issue when upgrading 
+   to Ceph 19.2.0 that requires one to disable the balancer in order for expected 
+   cluster operations. A fix has been implemented in 19.2.1, please read 
```

Comment:
I think "in order for expected cluster operations" should be changed to "in order for the cluster to operate normally". It seems almost like a garden path sentence as it is currently phrased.